core-web/libs/sdk/angular/src/lib/layout/container/container.component.ts

```diff
@@ -0,0 +1,94 @@
+import { AsyncPipe, NgComponentOutlet } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    HostBinding,
+    Input,
+    OnChanges,
+    computed,
+    inject,
+    signal
+} from '@angular/core';
+
+import { NoComponentComponent } from '../../components/no-component/no-component.component';
+import { DotCMSContainer, DotCMSContentlet, DynamicComponentEntity } from '../../models';
+import { PageContextService } from '../../services/dotcms-context/page-context.service';
+import { getContainersData } from '../../utils';
+
+interface DotContainer {
+    acceptTypes: string;
+    identifier: string;
+    maxContentlets: number;
+    uuid: string;
+    variantId?: string;
+}
+
+const EMPTY_CONTAINER_EDIT_MODE_STYLES = {
+    width: '100%',
+    backgroundColor: '#ECF0FD',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#030E32',
+    height: '10rem'
+};
+
+@Component({
+    selector: 'dotcms-container',
+    standalone: true,
+    imports: [AsyncPipe, NgComponentOutlet, NoComponentComponent],
+    templateUrl: './container.component.html',
+    styles: '',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ContainerComponent implements OnChanges {
```

Comment:
This component, has to have any docs explaining what and why exists?.

Reply:
This component no, only the LayoutComponent is exposed